2021q3/portmgr.adoc

```diff
@@ -0,0 +1,31 @@
+=== Ports Collection
+
+Links: +
+link:https://www.FreeBSD.org/ports/[About FreeBSD Ports] URL:link:https://www.FreeBSD.org/ports/[https://www.FreeBSD.org/ports/] +
+link:https://docs.freebsd.org/en/articles/contributing/ports-contributing/[Contributing to Ports] URL: link:https://docs.freebsd.org/en/articles/contributing/ports-contributing/[https://docs.freebsd.org/en/articles/contributing/ports-contributing/] +
+link:http://portsmon.freebsd.org/[FreeBSD Ports Monitoring] URL: link:http://portsmon.freebsd.org/[http://portsmon.freebsd.org/] +
+link:https://www.freebsd.org/portmgr/[Ports Management Team] URL: link:https://www.freebsd.org/portmgr/[https://www.freebsd.org/portmgr/] +
+link:http://ftp.freebsd.org/pub/FreeBSD/ports/ports/[Ports Tarball] URL: link:http://ftp.freebsd.org/pub/FreeBSD/ports/ports/[http://ftp.freebsd.org/pub/FreeBSD/ports/ports/]
+
+Contact: René Ladan <portmgr-secretary@FreeBSD.org> +
+Contact: FreeBSD Ports Management Team <portmgr@FreeBSD.org>
+
+The Ports Management Team is responsible for overseeing the overall direction of the Ports Tree, building packages, and personnel matters.
+Below is what happened in the last quarter.
+
+There are currently 46,500 ports in the Ports Tree according to FreshPorts.
+The open PR count for ports is currently 2,588, of which 608 are unassigned.
+The last quarter saw 9,833 commits on the "main" branch by 148 committers and 743 commits on the quarterly branch by 54 committers.
+Compared to 2021q2, activity mostly remained the same, the PR counts increased a bit but there were also more commits to the quarterly branch.
+
+During 2021q3, we welcomed Daniel Engberg (diizzy@) and Yasuhiro Kimura (yasu@), we said goodbye to culot@ and grog@ after their commit bits were taken in for safekeeping.
+
+Two new uses were introduced: angr to help with testing Python-related ports and mlt to help depending on mlt, a multimedia framework for TV broadcasting.
+Ruby saw minor updates for the 2.6, 2.7, and 3.0 series.
+
+The "big" packages were also updated: pkg to 1.17.2, Firefox to 93.0, and Chromium to 92.0.4515.159.
+
+As usual, exp-runs were performed by antoine@ but only those of July were recorded.
+There were 5 exp-runs to test adding CLOCK_*_COARSE compatibility definitions for Linux and to test ports updates.
```

Comment:
Markdown seems to parse the * as a formatting marker. If AsciiDoc does the same, you need to escape it.